lib/commands/team.js

```diff
@@ -0,0 +1,229 @@
+/* eslint-disable standard/no-callback-literal */
+
+const figgyPudding = require('figgy-pudding')
+const libteam = require('libnpmteam')
+const log = require('npmlog')
+const npmConfig = require('../config.js')
+
+const Team = module.exports = {
+  command: 'team',
+  describe: 'Used to manage teams in organizations, and change team memberships. Does not handle permissions for packages.',
+  aliases: ['t'],
+  builder (y) {
+    return y.help().alias('help', 'h')
+      .options(Team.options)
+      .demandCommand(1, 'Team subcommand is required')
+      .recommendCommands()
+      .command({
+        command: 'create',
+        aliases: ['c'],
+        describe: 'Create a new team',
+        builder: (y) => y.help().alias('help', 'h').options({
+          'scope': {
+            alias: 's',
```

Comment:
Can you remove all the aliases for _options_ from these commands? I'm fine with the command aliases themselves.

Reply:
Already removed :+1: